web/viewer.css

```
@@ -487,14 +489,55 @@ select {
   background-color: rgba(255, 102, 102, 1);
 }
 
-.secondaryToolbar {
+.secondaryToolbar,
+.editorParamsToolbar {
   padding: 6px 0 10px;
   inset-inline-end: 4px;
   height: auto;
   z-index: 30000;
   background-color: var(--doorhanger-bg-color);
 }
 
+#editorParamsToolbarContainer > .editorParamsSetter {
+  max-width: 220px;
+  min-height: 26px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-inline-start: 12px;
+  padding-inline-end: 12px;
```

Looking at https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline, this could be simplified to just:
```suggestion
  padding-inline: 12px;
```